### DISCUSSION TITLE
Timer issue

### DISCUSSION THREAD

**Discussion by punas123** [proginosko/LeechBlockNG#416] [Q&A] [answered]
Hi, i encountered an issue with the extension after trying to manually delete the .xpi file in the extensions folder in firefox/profiles/...

The timer has stopped working properly. It no longer shows up in the upper left corner (despite the option being activated both in the general options and blockset options) and when the timer reaches 0 (i can use the statistics page to see the timer), it doesn't immediately send me to the blockpage. It only sends me to the blockpage after i reload the tab.

I tried reinstalling leechblock and disabling other extensions i have, but nothing works.

Should i just reinstall firefox, or is there a less extreme way to solve the issue?

**Comment by proginosko** [comment, marked as answer]
Go to **about:addons**, click on LeechBlock NG, and select the Permissions tab. Is "Access your data for all websites" selected?

**Reply by punas123** [reply]
I wasn't, i turned it on and now it works. Thanks

**Reply by cyberpencake** [reply]
Thank you. This issue helped me as well.
